backend/lib/routes/members.js

```diff
@@ -66,3 +66,15 @@ router.route('/:name')
       next(err)
     }
   })
+
+router.route('/:name/secret')
```

Comment:
there is no route get secret. I would use verb `post` on the member route.

Reply:
done